content/v1.11/getting-started/provider-aws-part-2.md

```diff
@@ -741,12 +747,19 @@ my-second-composite-resource-nsz6j   True    True     my-second-composite-resour
 Because the _composite resource_ is the `Owner` of the _managed resources_, when
 Crossplane deletes the _composite resource_, it also deletes the _managed resources_ automatically.
 
-Delete the new _composite resource_ with `kubectl delete xdatabase`.
+Delete the new _composite resource_ with `kubectl delete composition`.
 
+Delete the first composition
 ```shell
-kubectl delete xdatabase my-second-composite-resource
+kubectl delete composition dynamodb-with-bucket
```

Comment:
In #271, @negz updated the `kubectl delete` commands:

* `kubectl delete composite my-second-composite-resource` to `kubectl delete xdatabase my-second-composite-resource`. 
* `kubectl delete claim claimed-database -n test` to `kubectl delete database claimed-database -n test`

Here, you are updating `kubectl delete xdatabase my-second-composite-resource` to 
`kubectl delete composition dynamodb-with-bucket`.

So apparently `xdatabase`, `composite`, and `composition` can be used to delete the same resources? This is confusing. It would be beneficial if the team standardized on a convention to interact with Kubernetes objects.

Reply:
I pushed another change to make this consistent and add a note that you can use `kubectl {} composite` or `kubectl {} <kind>`.

Reply:
Example: https://deploy-preview-360--crossplane.netlify.app/v1.11/getting-started/provider-aws-part-2/#verify-the-composite-resource